localboot/main.go

```diff
@@ -134,8 +135,7 @@ func BootGrubMode(devices []storage.BlockDev, baseMountpoint string, guid string
 	for _, cfg := range bootconfigs {
 		debug("Trying boot configuration %+v", cfg)
 		if err := cfg.Boot(); err != nil {
-			log.Printf("Failed to boot kernel %s: %v", cfg.Kernel.Name(), err)
-			cfg.Close()
```

Comment:
?

Reply:
this is not necessary anymore, the fields are all strings now, not `os.File`, and the cleanup is done in `BootConfig.Boot`

Reply:
okay